.github/workflows/test.yml

```diff
@@ -17,10 +17,9 @@ jobs:
           - macos
           - windows
         ruby:
-          - "2.5"
-          - "2.6"
-          - "2.7"
```

Comment:
Could you revert this?
We want to keep working with them as much as possible.

Reply:
Updated, it now uses Ruby versions 2.5 up to 3.2 (and head)